solcore/optics/rcwa.py

```diff
@@ -5,12 +5,24 @@
 import types
 from solcore.state import State
 
+
 rcwa_options = State()
 rcwa_options.size = [500, 500]
 rcwa_options.orders = 4
 rcwa_options.theta = 0
 rcwa_options.phi = 0
 rcwa_options.pol = 'u'
+rcwa_options.parallel = False
+rcwa_options.n_jobs = -1
+rcwa_options.rcwa_options = dict(LatticeTruncation='Circular',
```

Comment:
These options are repeated also above. Is there no way of setting them only once in one place rather than in two separate places?